sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/IntervalWindow.java

```diff
@@ -186,8 +186,7 @@ public boolean consistentWithEquals() {
 
     @Override
     public boolean isRegisterByteSizeObserverCheap(IntervalWindow value) {
-      return instantCoder.isRegisterByteSizeObserverCheap(value.end)
-          && durationCoder.isRegisterByteSizeObserverCheap(new Duration(value.start, value.end));
+      return true;
```

Comment:
should we inline the DurationCoder? It is just varint and we could then avoid creating Duration objects